app/controllers/project/settings/donations.js

```diff
@@ -76,8 +83,15 @@ export default Controller.extend({
      */
     saveDonationGoal(donationGoal, properties) {
       donationGoal.setProperties(properties);
+      console.log('set');
```

Comment:
Maybe we ought to explicitly sort the list in some way, then. By ID, maybe? Last updated? Amount?